src/main/java/com/minecolonies/coremod/commands/citizencommands/KillCitizenCommand.java

```diff
@@ -4,6 +4,7 @@
 import com.minecolonies.coremod.colony.CitizenData;
 import com.minecolonies.coremod.colony.Colony;
 import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.commands.IActionCommand;
 import com.minecolonies.coremod.entity.EntityCitizen;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.entity.EntityCitizen'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)